accounts-db/store-tool/src/main.rs

```diff
@@ -1,80 +1,56 @@
 use {
-    clap::{crate_description, crate_name, value_t, value_t_or_exit, App, Arg},
-    log::*,
-    solana_accounts_db::{
-        account_storage::meta::StoredAccountMeta, accounts_hash::AccountHash, append_vec::AppendVec,
-    },
-    solana_sdk::{
-        account::{AccountSharedData, ReadableAccount},
-        hash::Hash,
-        pubkey::Pubkey,
-    },
-    std::mem::ManuallyDrop,
+    clap::{crate_description, crate_name, value_t_or_exit, App, Arg},
+    solana_accounts_db::append_vec::AppendVec,
+    solana_sdk::{account::ReadableAccount, system_instruction::MAX_PERMITTED_DATA_LENGTH},
+    std::{mem::ManuallyDrop, num::Saturating},
 };
 
 fn main() {
-    solana_logger::setup_with_default_filter();
```

Comment:
Any specific reason for ripping this out? I guess we're not using too much solana stuff so maybe not any useful logs to be emit. FWIW, `ledger-tool` does have the same line:
https://github.com/anza-xyz/agave/blob/f43779bf05e61c886e53871f3daf8afc7ba62bfc/ledger-tool/src/main.rs#L556

Reply:
> Maybe that changes in the future and we re-add this?

👍 

> Also, for user-facing tools, I believe the correct/better UX is to not use log, and instead use the standard println/eprintln.

I'd agree with that. In the case of ledger-tool, the logs emitted by commands like `verify` are useful and often desired by whoever is running it (at the expense of noisy output)